_posts/2017-08-01-emldown.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "emldown: From machine readable EML metadata to a pretty documentation website"
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+  - name: Andrew MacDonald
+    url: http://aammd.info/
+  - name: Kara Woo
+    url: http://karawoo.com/
+  - name: Carl Boettiger
+    url: http://www.carlboettiger.info/
```

Comment:
@maelle can remove the url for Carl since our jekyll template links him to our `/staff` page